fastapi_jsonapi/atomic/schemas.py

```diff
@@ -183,7 +182,7 @@ def validate_operation(cls, values: dict):
 class AtomicOperationRequest(BaseModel):
     operations: List[AtomicOperation] = Field(
         alias="atomic:operations",
-        min_items=1,
+        min_length=1,
```

Comment:
does this work as intended? I had red tests after this change

Reply:
`min_items` is deprecated and removed. https://docs.pydantic.dev/2.6/migration/#changes-to-pydanticfield